src/test/java/com/sfuentes/EmployeeTest.java

```diff
@@ -0,0 +1,84 @@
+package com.sfuentes;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class EmployeeTest {
+
+  @Test
+  void test_exceptionIncorrectAmountOfSales() {
+    double[] sales = {100.00, 100.00, 100.00, 100.00};
+
+    Exception exception = assertThrows(IllegalArgumentException.class, () -> {
+      Employee test = new Employee(1, sales);
+    });
+
+    String expectedMessage = "Incorrect amount of sales";
+    String actualMessage = exception.getMessage();
+
+    assertTrue(actualMessage.contains(expectedMessage));
+  }
+
+  @Test
+  void calculateSumSalary_integer() {
+    double[] sales = {100.00, 100.00, 100.00};
+    Employee test = new Employee(1, sales);
+    double expected = 30.00;
+    double actual = test.calculateSumSalary();
+
+    assertEquals(expected, actual);
```

Comment:
Here is better if you put the literal value instead of expected, also put the method call directly instead of storing it in a variable, it will be more readable when taking a look at the assertion and please do the same to the other assertions.

assertEquals(30d, test.calculateSumSalary());

Reply:
fixed